### ISSUE TITLE
Setting speaker availibilty to "all day" for the last day of the conference triggers an error message of "Submitted availability is not within the event timeframe."

### ISSUE THREAD

**Issue by adengg** [pretalx/pretalx#460]
## Expected Behavior
Setting the availibilty to "All Day" should be possible for all days of the conference, even if it ends at some point during this day.

## Current Behavior
Setting the availibility to "All Day" on all days of the conference (in our case saturday to sunday) triggers an error message of "Submitted availability is not within the event timeframe."

## Steps to Reproduce

1. Set the conference date to start at 2018-10-22 and end at 2018-10-28
2. Edit a speaker
3. Set the availability to all day for all conference days
4. On the bottom of the page, you the the error message "Submitted availability is not within the event timeframe."

if needed, we can provide a speaker account on the instance for testing.

## Your Environment
* Version used: 0.7.1
* Environment name and version (e.g. Chrome 39, python 3.5): Firefox 61
* Operating System and version (desktop or mobile): Linux Desktop (Debian Unstable)
* Link to your instance, if in production: https://conference.c3w.at

**Comment by rixx** [comment]
Good catch, thank you for the report! Definitely a bug – I may take a few days to look into it, since I'm travelling rn, but I'll issue a bugfix release once it's fixed.

**Comment by luto** [comment]
haha, well, the last day of privacyweek is the day Austria switches back to standard time.. soo, well, [timezones](https://youtu.be/-5wpm-gesOY) ;) m) I will take a deeper look tomorrow, just know that this only affects a single day or two a year.

**Comment by rixx** [comment]
A single day, most likely, as it's only affecting shortened days.
